manage.py

```diff
@@ -32,6 +32,37 @@
 manager.add_command('runserver', Server(use_debugger=True, use_reloader=True))
 
 
+def check_itemized_queues(schedule):
+    """Checks to see if the queues associated with an itemized schedule have
+    been successfully cleared out and sends the information to the logs.
+    """
+
+    remaining_new_queue = db.engine.execute(
+        'select count(*) from ofec_sched_{schedule}_queue_new'.format(
+            schedule=schedule
+        )
+    ).scalar()
+    remaining_old_queue = db.engine.execute(
+        'select count(*) from ofec_sched_{schedule}_queue_old'.format(
+            schedule=schedule
+        )
+    ).scalar()
+
+    if remaining_new_queue == remaining_old_queue == 0:
```

Comment:
Hrm, you're right, technically speaking they could...  Okay, I will think on this a bit more.  That raises another question though:  the existing process would update everything and then just wholesale delete everything when done (and this will still be the case for Schedule E), so in that event, we would've been losing updates there as well.

I'm going to assume table locks aren't desired and rule them out right now, but I'll mull this over/read up on some things to see if there's a better way.

Thanks for catching and flagging this!